Enhancements Developer Section

hauni97 opened the pull request:
Minor improvements.

hauni97 commented:
Only thing missing is a search-bar in the getting started section. @limichange if you wish to help.

limichange commented:
![image](https://user-images.githubusercontent.com/1947344/146495921-61ea4541-eafd-432b-b526-3915e11f46b5.png) 
The `doks` has used the `algolia` to search all docs, should we update it too?
@hauni97 @h4x3rotab

limichange commented:
If we want to do it, we need to create a `algolia` account and get a development code.

h4x3rotab commented:
> If we want to do it, we need to create an `algolia` account and get a development code.

Sounds perfect. Could you help with "dev@phala.network" to register one account?

limichange commented:
Adding a new search feature doesn't seem to be that simple, so I'll open a new PR to deal with it. So this branch can be left alone for now for search-related issues.

h4x3rotab commented:
Is it ready to review?

hauni97 commented:
> Is it ready to review?

Yes, please review @h4x3rotab.

https://deploy-preview-49--phala-wiki-preview.netlify.app

hauni97 commented:
# Changes in Wiki

## Some of the new improvements:

- [x] New sections & separate pages for General, Learn, Build, Maintain and Getting Started. As they are on separate pages, each required new HTML sections for the layout & sidebar: https://github.com/Phala-Network/phala-wiki-next/pull/49/commits/50f40911b45e2ac7573e3acdfe2032c5aec31d0b https://github.com/Phala-Network/phala-wiki-next/pull/49/commits/ced36ebe41c6d030980d3a28649af4e52dd48204 https://github.com/Phala-Network/phala-wiki-next/pull/49/commits/314bc91220a36a8ca1bcf8d57ba28ef2860730d0 https://github.com/Phala-Network/phala-wiki-next/pull/49/commits/e716f55e539d665027e6545504d1dd9268f25a5e https://github.com/Phala-Network/phala-wiki-next/pull/49/commits/6f03605beb5493f13a2050e026ed02c100238c70 https://github.com/Phala-Network/phala-wiki-next/pull/49/commits/a64b50a51de8c5284e3d023df559371b8831c49d & more...
- [x] Expandable Menu for mobile devices added via HTML headers: https://github.com/Phala-Network/phala-wiki-next/pull/49/commits/7b2f36429afae15c3269e086f7037051bb225a09
- [x] Main & 404 error was switched to General -> Learn: https://github.com/Phala-Network/phala-wiki-next/commit/e716f55e539d665027e6545504d1dd9268f25a5e
- [x] Browser Extension Guide under General -> Phala Wallet has new content (explaining intro paragraph as desired by @shelvenzhou): https://github.com/Phala-Network/phala-wiki-next/pull/49/commits/a9e6b1a351aa832e4eedf5362e08c2940c4cf8ae 
    * Improved readability by scaling the images & adjusting headings, as well as grammar: https://github.com/Phala-Network/phala-wiki-next/pull/49/commits/cfcbb74bc02ed91675849d5d75b84a1651e346ca
- [x] Gatekeeper under learn -> Maintain, was added with new content (intro paragraph) and remaining information was rewritten (content is subject to review to determine if it is up to date): https://github.com/Phala-Network/phala-wiki-next/pull/49/commits/cc3a3f5cd199ace8fe8c18044fae069777cdf1e1
- [x] Phala Blockchain in Detail -> Learn: Renamed headings & added content to emphasise @shelvenzhou's points: https://github.com/Phala-Network/phala-wiki-next/pull/49/commits/f2f31c6ba10aecc28c9e8bc77e3257fa8c8da809
- [x] Plenty of additional edits & improvements: [100+ commits](https://github.com/Phala-Network/phala-wiki-next/pull/49/commits) in total

## Issues (3)

- [ ] Missing "Search Bar", not global; it is page agnostic (currently only works in “maintain”)
- [ ] Maintain -> Gatekeeper: section is not expanding and showing its articles (not solved yet)
- [ ] Buttons for next page (at the bottom of each article) are not working as intended. Reported here: https://github.com/Phala-Network/phala-wiki-next/issues/60 
- _Hypothesis of related files:_ https://github.com/Phala-Network/phala-wiki-next/blob/8ee0e3819978b95cd81560bbb965f51d8c7bbcf9/layouts/partials/main/docs-navigation.html#L20-L31
 https://github.com/Phala-Network/phala-wiki-next/blob/8ee0e3819978b95cd81560bbb965f51d8c7bbcf9/assets/scss/common/_global.scss#L182-L201

## TODO 

- [ ] Blockchain Entities: + Video Tutorial: share resources | @HashWarlock I would appreciate you sharing the links. 
- [ ] + Getting Started -> Stake Phala image of brokers missing (should be added) + content
- [ ] Wallet Identity: /en-us/general/phala-wallet/set-up-identity/ cosmetics (resize images for conciseness)
- [ ] Add Scores of miners under Tokenomics (from Server)
- [ ] Cleanup of: 3.0 Console.md: /en-us/docs/khala-mining/3-0-console/ [here](https://deploy-preview-49--phala-wiki-preview.netlify.app/en-us/docs/khala-mining/3-0-console/)
- [ ] General content check of all pages (is it up to date?) @shelvenzhou please let me know
- [ ] Merge PR: Publish new wiki :tada:

@h4x3rotab @shelvenzhou please review the current status

hauni97 commented:
> # Changes in Wiki
> Preview 👉 https://deploy-preview-49--phala-wiki-preview.netlify.app/en-us/learn/products/about-pdiem
> 
> ## Some of the new improvements:
> * [x]  New sections & separate pages for General, Learn, Build, Maintain and Getting Started. As they are on separate pages, each required new HTML sections for the layout & sidebar: [50f4091](https://github.com/Phala-Network/phala-wiki-next/commit/50f40911b45e2ac7573e3acdfe2032c5aec31d0b) [ced36eb](https://github.com/Phala-Network/phala-wiki-next/commit/ced36ebe41c6d030980d3a28649af4e52dd48204) [314bc91](https://github.com/Phala-Network/phala-wiki-next/commit/314bc91220a36a8ca1bcf8d57ba28ef2860730d0) [e716f55](https://github.com/Phala-Network/phala-wiki-next/commit/e716f55e539d665027e6545504d1dd9268f25a5e) [6f03605](https://github.com/Phala-Network/phala-wiki-next/commit/6f03605beb5493f13a2050e026ed02c100238c70) [a64b50a](https://github.com/Phala-Network/phala-wiki-next/commit/a64b50a51de8c5284e3d023df559371b8831c49d) & more...
> * [x]  Expandable Menu for mobile devices added via HTML headers: [7b2f364](https://github.com/Phala-Network/phala-wiki-next/commit/7b2f36429afae15c3269e086f7037051bb225a09)
> * [x]  Main & 404 error was switched to General -> Learn: [e716f55](https://github.com/Phala-Network/phala-wiki-next/commit/e716f55e539d665027e6545504d1dd9268f25a5e)
> * [x]  Browser Extension Guide under General -> Phala Wallet has new content (explaining intro paragraph as desired by @shelvenzhou): [a9e6b1a](https://github.com/Phala-Network/phala-wiki-next/commit/a9e6b1a351aa832e4eedf5362e08c2940c4cf8ae)
>   
>   * Improved readability by scaling the images & adjusting headings, as well as grammar: [cfcbb74](https://github.com/Phala-Network/phala-wiki-next/commit/cfcbb74bc02ed91675849d5d75b84a1651e346ca)
> * [x]  Gatekeeper under learn -> Maintain, was added with new content (intro paragraph) and remaining information was rewritten (content is subject to review to determine if it is up to date): [cc3a3f5](https://github.com/Phala-Network/phala-wiki-next/commit/cc3a3f5cd199ace8fe8c18044fae069777cdf1e1)
> * [x]  Phala Blockchain in Detail -> Learn: Renamed headings & added content to emphasise @shelvenzhou's points: [f2f31c6](https://github.com/Phala-Network/phala-wiki-next/commit/f2f31c6ba10aecc28c9e8bc77e3257fa8c8da809)
> * [x]  New Fat Contract Content in Learn: https://deploy-preview-49--phala-wiki-preview.netlify.app/en-us/learn/developer/fat-contract/
> * [x]  New Fat Contract Content in [Build](https://deploy-preview-49--phala-wiki-preview.netlify.app/en-us/build/developer/fat-contract-examples/): [435e158](https://github.com/Phala-Network/phala-wiki-next/commit/435e15844fc69aca2e2bf9b6168cba83979cfb3a)
> * [x]  Content improvement for /en-us/general/phala-wallet/set-up-identity/: [60b6011](https://github.com/Phala-Network/phala-wiki-next/commit/60b60112a60f88871e7cc7dd4437ba61d91cbf99)
> * [x]  This PR contains a brand new "[Getting Started](https://deploy-preview-49--phala-wiki-preview.netlify.app/en-us/getting-started/stake-khala/stake-phala/)" section. Latest commit: [a09bd7e](https://github.com/Phala-Network/phala-wiki-next/commit/a09bd7efad16dce3a507a53d4baf0ede606659ed)
> * [x]  Plenty of additional edits & improvements in Mining & Staking: [100+ commits](https://github.com/Phala-Network/phala-wiki-next/pull/49/commits) in total


* [x] Mining + Staking article polishing(create a GIF for the aspect)
    * Consisted of redrawing & modification of images (+scenarios) to creating a GIF:https://github.com/Phala-Network/phala-wiki-next/pull/49/commits/40707f873d36fa1d3b8a082c0312f5b18485a3c8 https://github.com/Phala-Network/phala-wiki-next/pull/49/commits/a1b3dd1516c57a6ce3f6ec08588ad9f65ed46a01 https://github.com/Phala-Network/phala-wiki-next/pull/49/commits/16bc0fec11c31019d120519513fbc2c0fe692fce 
    * As a result the readability is improved, due to a cleaner page
    * Added the [how to video](https://deploy-preview-49--phala-wiki-preview.netlify.app/en-us/learn/tokenomics/mining-staking/#how-to-reclaim): https://github.com/Phala-Network/phala-wiki-next/pull/49/commits/c6bb2c1dd7958ad02b14d895909e92e4dc9c3fc6 
    * Content Restructuring: https://github.com/Phala-Network/phala-wiki-next/pull/49/commits/c23ff4e946a54870b5d7e017c474851b9b7ad33d
    * Plenty of other improvements: https://github.com/Phala-Network/phala-wiki-next/pull/49/commits/a3e38d012b030a0202b456b9ca3f90de712c670b https://github.com/Phala-Network/phala-wiki-next/pull/49/commits/cdb85a0ecbc2eac62798a4540c1b4a39bfb6cc1b  https://github.com/Phala-Network/phala-wiki-next/pull/49/commits/dbd6f252409689fa07e1927707a647f5893f4381 https://github.com/Phala-Network/phala-wiki-next/pull/49/commits/ed2d6e52e6659c0652d4710455484b4fe43e2cad 
* [x] Added a Video for Getting Started: https://github.com/Phala-Network/phala-wiki-next/pull/49/commits/90fcbbac78c112bcca001ae5612a91689eddd3ea
> 
> ## Issues (3)
> * [ ]  Missing "Search Bar", not global; it is page agnostic (currently only works in “maintain”)
> * [ ]  Maintain -> Gatekeeper: section is not expanding and showing its articles (not solved yet)
> * [ ]  Buttons for next page (at the bottom of each article) are not working as intended. Reported here: [docs-navigation buttons in PR #49 #60](https://github.com/Phala-Network/phala-wiki-next/issues/60)
> * _Hypothesis of related files:_ https://github.com/Phala-Network/phala-wiki-next/blob/8ee0e3819978b95cd81560bbb965f51d8c7bbcf9/layouts/partials/main/docs-navigation.html#L20-L31
>   
>   https://github.com/Phala-Network/phala-wiki-next/blob/8ee0e3819978b95cd81560bbb965f51d8c7bbcf9/assets/scss/common/_global.scss#L182-L201
> 
> ## TODO
> * [x]  Blockchain Entities: + Video Tutorial: share resources | @HashWarlock I would appreciate you sharing the links.
> * [x]  + Getting Started -> Stake Phala image of brokers missing (should be added) + content
> * [x]  Wallet Identity: /en-us/general/phala-wallet/set-up-identity/ cosmetics (resize images for conciseness)
 * [x]  Add Scores of miners under Tokenomics (from Server) -> Xeon was added: https://github.com/Phala-Network/phala-wiki-next/pull/49/commits/1591153214ea6af17f2e39de4f0112b859ea5d19
* [x]  Cleanup of: 3.0 Console.md: /en-us/docs/khala-mining/3-0-console/ [here](https://deploy-preview-49--phala-wiki-preview.netlify.app/en-us/docs/khala-mining/3-0-console/): https://github.com/Phala-Network/phala-wiki-next/pull/49/commits/90fcbbac78c112bcca001ae5612a91689eddd3ea 
> * [ ]  General content check of all pages (is it up to date?) @shelvenzhou please let me know
> * [ ]  Merge PR: Publish new wiki 🎉
> * [ ]  Check where links need to be updated (Discord etc.)
> 
> @h4x3rotab @shelvenzhou please review the current status

shelvenzhou reviewed (approved):
LGTM
